src/command_modules/azure-cli-network/azure/cli/command_modules/network/tests/test_network_commands.py

```diff
@@ -205,13 +199,13 @@ def body(self):
         self.cmd('network express-route circuit list --resource-group {}'.format(self.resource_group),
             checks=NoneCheck())
 
-class NetworkExpressRouteCircuitScenarioTest(VCRTestBase):
+class NetworkExpressRouteCircuitScenarioTest(ResourceGroupVCRTestBase):
 
     def __init__(self, test_method):
          # The resources for this test did not exist so the commands will return 404 errors.
          # So this test is for the command execution itself.
```

Comment:
@tjprescott yeah, we should create an issue for this one

---
In reply to: [78781142](https://github.com/Azure/azure-cli/pull/875#discussion_r78781142) [](ancestors = 78781142)